src/webhook/mutation/pod_mutator/oneagent_mutation/versionlabel.go

```diff
@@ -18,11 +26,26 @@ func newVersionLabelMapping(namespace corev1.Namespace) VersionLabelMapping {
 }
 
 func getMappingFromNamespace(namespace corev1.Namespace) VersionLabelMapping {
-	// TODO: Implementation => Parse Namespace annotations into correct format
-	return VersionLabelMapping{}
+	mappingKey2versionLabelKey := map[string]string{
+		versionMappingAnnotationName: releaseVersionEnv,
+		productMappingAnnotationName: releaseProductEnv,
+		stageMappingAnnotationName:   releaseStageEnv,
+		buildVersionAnnotationName:   releaseBuildVersionEnv,
+	}
+
+	versionLabelMapping := VersionLabelMapping{}
+	for mappingKey, versionLabelKey := range mappingKey2versionLabelKey {
```

Comment:
```suggestion
	for annotation, label := range mappingKey2versionLabelKey {
```

Reply:
Both annotation and label conceptually are key and value pairs.
In this case, we assign `...Key` as a value and I think keeping naming straightforward is better.

Reply:
how about `annotationKey` and `labelKey`?